admin/src/pages/CreationConfirm.spec.js

```diff
@@ -89,6 +91,11 @@ describe('CreationConfirm', () => {
     adminDeleteContributionMock.mockResolvedValue({ data: { adminDeleteContribution: true } }),
   )
 
+  mockClient.setRequestHandler(
+    denyContribution,
+    adminRejectContributionMock.mockResolvedValue({ data: { denyContribution: true } }),
```

Comment:
You could do something like:
```
mockClient.setRequestHandler(
    denyContribution,
    jest.fn().mockRejectedValueOnce({ message: 'Deny contribution failed.' })
      .mockResolvedValue({ data: { denyContribution: true } }),
}
```
To test the error. You must test the error first